python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -4110,3 +4110,61 @@ def test_cov_nans():
     gdf = gd.from_pandas(pdf)
 
     assert_eq(pdf.cov(), gdf.cov())
+
+
+@pytest.mark.parametrize(
+    "gsr",
+    [
+        Series([1, 2, 3]),
+        Series([1, 2, 3], index=["a", "b", "c"]),
+        Series([1, 2, 3], index=["a", "b", "d"]),
+        Series([1, 2], index=["a", "b"]),
+        Series([1, 2, 3], index=gd.core.index.RangeIndex(0, 3)),
+    ],
+)
+@pytest.mark.parametrize("colnames", [["a", "b", "c"], [0, 1, 2]])
+def test_df_sr_binop(gsr, colnames):
+    data = [[0, 1, 2], [3, None, 5], [6, 7, np.nan]]
+    data = dict(zip(colnames, data))
+
+    gdf = DataFrame(data)
+    pdf = pd.DataFrame.from_dict(data)
+
+    psr = gsr.to_pandas()
+
+    # add
```

Comment:
Can we make the op a test parameter so that failures are more easily triaged?

Reply:
added this - lots of tests pop over other ops such as `pow`, looking into it.